packages/webview_flutter/webview_flutter/example/lib/main.dart

```diff
@@ -15,6 +15,7 @@ import 'package:webview_flutter/webview_flutter.dart';
 // #docregion platform_imports
 // Import for Android features.
 import 'package:webview_flutter_android/webview_flutter_android.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
```

Comment:
Why do we need this? It feels like we're not re-exporting something we should be if this is necessary for basic usage.